docs/conf.py

```diff
@@ -10,7 +10,7 @@
 
 
 docs_dir = os.path.abspath(os.path.dirname(__file__))
-base_dir = os.path.join(docs_dir, "..")
+base_dir = os.path.abspath(os.path.join(docs_dir, ".."))
```

Comment:
As soon as all dependencies of genialis-base will run on Python 3 :P

I am already developing Resolwe on py3